images/docker-stacks-foundation/Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN apt-get update --yes && \
     bzip2 \
     ca-certificates \
     locales \
+    netbase \
```

Comment:
It would be nice to add a comment why we're adding this package

Reply:
> Add netbase so that `/etc/{protocols,rpc,services}` exists

– https://github.com/docker-library/python/commit/b8c94a31a98a535477200482a32c95192f85af5b

Reply:
No reason is given *why* it was added to the Docker "Official Image" for `python`.

Maybe @yosifkit can give some insight.

Reply:
It was to fix https://github.com/docker-library/python/issues/270, or to ensure that `socket.getservbyname()` works.

Reply:
Thanks! @AlexHill could you please add a comment to the code so the next person who sees it will know why it was added. It would be nice to add a link to this PR there as well.